playbooks/templates/rax-maas/cinder_volume_check.yaml.j2

```diff
@@ -2,6 +2,9 @@
 {% if (ansible_local['maas']['general']['deploy_osp'] | bool) | default(False) %}
 {% set cinder_volume_host = item.value.host %}
 {% set backend_name = cinder_volume_host.split('@')[1] %}
+{% elif (rbd_cinder_backend | default(False) | bool) and (item.key == 'ceph' or item.value.volume_backend_name == 'ceph') %}
```

Comment:
I don't  think we need `and (item.key == 'ceph' or item.value.volume_backend_name == 'ceph')` as the volume backend can be different, especially with multiple ceph pools and OSA just hard codes it once it sees the rbd driver enabled